src/services/ledger/receive.ts

```diff
@@ -109,18 +111,38 @@ export const receive = {
     walletId,
     walletCurrency,
     paymentHash,
-    amountDisplayCurrency,
     journalId,
     sats,
     cents,
     revealedPreImage,
+    paymentFlow,
+    feeDisplayCurrency,
+    amountDisplayCurrency,
+    displayCurrency,
   }: AddLnFeeReeimbursementReceiveArgs): Promise<LedgerJournal | LedgerError> => {
+    const {
+      btcPaymentAmount: { amount: satsAmount },
+      btcProtocolFee: { amount: satsFee },
+    } = paymentFlow
+    const centsAmount = Math.round((amountDisplayCurrency - feeDisplayCurrency) * 100)
+    const centsFee = Math.round(feeDisplayCurrency * 100)
```

Comment:
are we sure we want cents conversion responsibility in the ledger service? shouldn't be part of the price service?

Reply:
This is very temporary here... eventually gets handled by our payment flow builder ([here](https://github.com/GaloyMoney/galoy/pull/1237/files#diff-9152ae6f6a4bd5040a65b4bfa9eb5cd1ffadfc9c069ff3b39cd7dc78e02fde1dR128))